### DISCUSSION TITLE
Can I use it in vanila JS without JS framework?

### DISCUSSION THREAD

**Discussion by sajadabedi** [chakra-ui/ark#1714] [Q&A] [answered]
I'm building with Ruby on Rails and I'm wondering if it's possible to use Ark with vanilla JavaScript?

**Comment by cschroeter** [comment, marked as answer]
@sajadabedi 

Thanks for bringing this up. @segunadebayo and I already talked about this in the sense of supporting frameworks like https://htmx.org/

But to answer your question: At the moment this won't work out of the box but chances are good that we add support for vanilla js at some point.

**Reply by movahhedi** [reply]
Any news on Vanilla support?
